src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java

```diff
@@ -121,6 +138,22 @@ protected void writeColumnSpec(StringBuilder builder, SinkRecordField f) {
   }
 
   protected void formatColumnValue(StringBuilder builder, String schemaName, Map<String, String> schemaParameters, Schema.Type type, Object value) {
+    if (schemaName != null) {
+      switch (schemaName) {
+        case Decimal.LOGICAL_NAME:
+          builder.append(value);
```

Comment:
In some (maybe all?) databases, this could exceed the maximum value supported by the underlying type, right? e.g. in MySQL you have to map to a 65 bit maximum unscaled value. However, this isn't checking for that (nor are each of the dialect classes). Is this intentional and we just expect an error to be thrown, or did we just miss the check?